### ISSUE TITLE
Focus for user-menu nav is not accessible

### ISSUE THREAD

**Issue by emoral435** [nextcloud/server#42565]
# Issue Summary
Whenever a user tabs onto the selected view for their user-menu navbar, the contrast ratio may be too low.

EG. : 

![Image](https://github.com/nextcloud/server/assets/110193237/19a9546e-2f56-4afe-8e2c-8bd448cec472)

This may be to hard for many to see

**Comment by JuliaKirschenheuter** [comment]
😭 good catch!
